includes/LocalisationCache.php

```diff
@@ -457,8 +457,8 @@ protected function readPHPFile( $_fileName, $_fileType ) {
 		// wikia changes begin
 		//* default empty for wikia i18n files */
 		$messages = array();
+		include_once( $_fileName );
```

Comment:
Is this sorted out? I mean are we done with duplicates for good and `include_once` won't cause any trouble any more?